website/notifications/utils.py

```diff
@@ -404,6 +405,16 @@ def subscribe_user_to_notifications(node, user):
 
     :param user: User to subscribe to notifications
     """
+
+    if node.institution_id:
+        raise InvalidSubscriptionError('Institutions are invalid targets for subscriptions')
+
+    if node.is_bookmark_collection:
```

Comment:
Ok wasn't sure what the difference between bookmark_collections and collections. Are all collections invalid targets for subscriptions?